src/chapters/interfaces-and-polymorphism/index.rst

```diff
@@ -0,0 +1,10 @@
+Interfaces and Polymorphism
+============================
+
+.. toctree::
+   :maxdepth: 3
+
+   introduction
+   exercises
+   studio
+   new-studio
```

Comment:
Change ``new-studio`` to the ``studio`` and get rid of the old studio.

Reply:
Done!